Makefile.automated

```diff
@@ -1247,7 +1659,225 @@ datasets/lvwiki.autolabeled_revisions.w_cache.20k_2016.json: \
+		--verbose > $@
+
+datasets/nlwiki.labeled_revisions.w_cache.20k_2016.json: \
+		datasets/nlwiki.labeled_revisions.20k_2016.json
+	cat $< | \
+	revscoring extract \
+		editquality.feature_lists.nlwiki.damaging \
+		editquality.feature_lists.nlwiki.goodfaith \
+		--host https://nl.wikipedia.org \
+		--extractor $(max_extractors) \
+		--verbose > $@
+
+tuning_reports/nlwiki.damaging.md: \
+		datasets/nlwiki.labeled_revisions.w_cache.20k_2016.json
+	cat $< | \
+	revscoring tune \
+		config/classifiers.params.yaml \
+		editquality.feature_lists.nlwiki.damaging \
+		damaging \
+		roc_auc.labels.true \
+		--label-weight "true=$(damaging_weight)" \
+		--pop-rate "true=0.05068086413432989" \
+		--pop-rate "false=0.9493191358656701" \
+		--center --scale \
+		--cv-timeout 60 \
+		--debug > $@
+
+models/nlwiki.damaging.gradient_boosting.model: \
+		datasets/nlwiki.labeled_revisions.w_cache.20k_2016.json
+	cat $< | \
+	revscoring cv_train \
+		revscoring.scoring.models.GradientBoosting \
+		editquality.feature_lists.nlwiki.damaging \
+		damaging \
+		--version=$(damaging_major_minor).0 \
+		-p 'learning_rate=0.01' \
+		-p 'max_depth=5' \
+		-p 'max_features=log2' \
+		-p 'n_estimators=700' \
+		--label-weight "true=$(damaging_weight)" \
+		--pop-rate "true=0.05068086413432989" \
+		--pop-rate "false=0.9493191358656701" \
+		--center --scale > $@
+
+tuning_reports/nlwiki.goodfaith.md: \
+		datasets/nlwiki.labeled_revisions.w_cache.20k_2016.json
+	cat $< | \
+	revscoring tune \
+		config/classifiers.params.yaml \
+		editquality.feature_lists.nlwiki.goodfaith \
+		goodfaith \
+		roc_auc.labels.true \
+		--label-weight "false=$(goodfaith_weight)" \
+		--pop-rate "true=0.9646257806900789" \
+		--pop-rate "false=0.03537421930992113" \
+		--center --scale \
+		--cv-timeout 60 \
+		--debug > $@
+
+models/nlwiki.goodfaith.gradient_boosting.model: \
+		datasets/nlwiki.labeled_revisions.w_cache.20k_2016.json
+	cat $< | \
+	revscoring cv_train \
+		revscoring.scoring.models.GradientBoosting \
+		editquality.feature_lists.nlwiki.goodfaith \
+		goodfaith \
+		--version=$(goodfaith_major_minor).0 \
+		-p 'learning_rate=0.01' \
+		-p 'max_depth=5' \
+		-p 'max_features=log2' \
+		-p 'n_estimators=700' \
+		--label-weight "false=$(goodfaith_weight)" \
+		--pop-rate "true=0.9646257806900789" \
+		--pop-rate "false=0.03537421930992113" \
+		--center --scale > $@
+
+nlwiki_models: \
+	models/nlwiki.damaging.gradient_boosting.model \
+	models/nlwiki.goodfaith.gradient_boosting.model
+
+nlwiki_tuning_reports: \
+	tuning_reports/nlwiki.damaging.md \
+	tuning_reports/nlwiki.goodfaith.md
+
+############################# Portugueses Wikipedia ################################
+
+datasets/ptwiki.human_labeled_revisions.20k_2015.json:
+	./utility fetch_labels \
+		https://labels.wmflabs.org/campaigns/ptwiki/7/ > $@
+
+datasets/ptwiki.labeled_revisions.20k_2015.json: \
+		datasets/ptwiki.human_labeled_revisions.20k_2015.json
+	./utility merge_labels $^ > $@
```

Comment:
Another single-file merge

Reply:
Yeah, The weird thing with enwiki and ptwiki is that they directly retrieve labels from wikilabels and don't add anything. This is the only way I could think of :/